### ISSUE TITLE
Artifacting MBP 10,1

### ISSUE THREAD

**Issue by AndreasJordanidis** [dortania/OpenCore-Legacy-Patcher#980]
### Machine Model

MacBookPro10,1

### Application Version

Latest Release

### Application Variant

TUI (Text User Interface)

### Have you verified whether this issue has been opened before? If no, your issue will be closed

true

### Have you verified whether this issue is covered in our guide? ie. Troubleshooting and Legacy Acceleration pages

true

### Have you already verified this is a patcher bug and not a macOS bug/user error on our discord servers?

false

### Is this issue with a 3rd party application?

false

### What versions of macOS are you seeing the problem on?

macOS 12, Monterey

### Where does this issue happen?

Within macOS (After Logging in)

### What is the Issue?

Firstly, id like to apologize for not using discord, but I do not use that social network anymore.
Basically im getting artifacting in the rounded corners and launchpad folders, it looks pixelated compared to Big Sur and Catalina.
Is there a temporary fix for this? or is this unsolvable?

Ive seen that some users on supported Macs are also having this issue, it seems to only be present on iris powered Macs.

### What steps have you taken to troubleshoot this? Please include messages from Discord, logs, and screenshots.

changing resolution

**Comment by khronokernel** [comment]
> Ive seen that some users on supported Macs are also having this issue

You've answered your own question, if it's an Apple bug affecting native hardware there's nothing we can do

**Comment by AndreasJordanidis** [comment]
its only affecting iris Macs, which I find strange cuz even ivy bridge and kepler are affected.
